chapters/introduction.tex

```diff
@@ -1,11 +1,7 @@
 \section{Introduction} \label{sec:introduction}
-% Setting the scene for the project, giving it its context
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+With technology steadily improving over the years, people are becoming more interconnected than ever and entertainment in form of applications are plentiful. Despite this, depression continues to rise among the general population. According to the World Health Organization, over 322 million cases of depressive disorder were recorded in 2017, representing a rise of around 18.4\% over the past ten years \cite{who_depression}. Not only does depression negatively impact wellbeing, it may also lead to complications such as increased fatigue, decreased motivation or even suicide. As a result, researchers have used computing technologies known as \textit{Affective Computing} that seek to identify human emotions in order to combat mental disorders \cite{ieee_affective}.

```

Comment:
I'd suggest using more line breaks to make commenting easier

Reply:
I separated each sentence with a line break now. Should make it easier to review.